src/services/authentication/index.js

```diff
@@ -48,11 +50,31 @@ module.exports = function() {
 		passwordField: 'password'
 	};
 
+	const cookieExtractor = function(req) {
```

Comment:
Duplicate cookie-jwt-extractor-function, might result in inconsistent code if only one of them gets updated sometimes. Any way how to make that one "global" function?